src/modules/endpoint/DataContainer.js

```diff
@@ -30,83 +24,83 @@ export interface DataContainer {
    * Returns an Array of LanguageModels.
    * @throws Will throw an error if the array is null.
    */
-  getLanguages: () => Promise<Array<LanguageModel>>,
+  getLanguages: (context: DatabaseContext) => Promise<Array<LanguageModel>>,
```

Comment:
Context makes it compatible with the new caching mechanism.

Reply:
Does this mean if you try to get the language with a different context, i.e. a different language, this will return null even though it should be the same? E.g. if you set the languages with context `augsburg, de` and load the languages with context `augsburg en`?.

Reply:
Same for `getCities`

Reply:
Yes, you are right, the languages depend now not only on the city, but also on the language

Reply:
But that doesn't make sense... that way we have to load it separately every time, don't we?

Reply:
Langauges actually depend on the city and on the language. So any change in the context means a fetch is required.

Reply:
Hm weird it loads an url like: https://cms.integreat-app.de/augsburg/de/wp-json/extensions/v3/languages

Reply:
Yep, this is an cms issue. We always load the languages with `de` because `de` always exists, that's why you don't have to pass a language to the languages endpoint. So fetching the languages for every language is just unnecessary. Same goes for `getCities`

Reply:
Hm I don't think this is a cms issue after all. For example if the cms would be configured right:

`augsburg/de/wp-json/extensions/v3/languages` would return "Englisch"
`augsburg/en/wp-json/extensions/v3/languages` would return "English"

Reply:
No this is something we definitely decided on a few year(s) ago! We always load 'de' since they might even return inconsistent language sets and the one configured in the `de`-version is always correct! Besides, the "title" of the language is always translated in the Native Language itself...

Reply:
Hm yeah not sure. Long time, maybe a commit message documents this but I'm not sure ^^

But imo it makes no sense what we decided earlier. I also communicated the Django team half a year ago that the CMS delivers the languages based on the current language.

**For this PR, I would make it depend only on the city because of the api-client.**